### PR TITLE
restructure mistral examples and add readme

### PR DESCRIPTION
**Pull request by rishiraj**

### CODE REVIEW

File: docs/docs/examples/mistral/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Indexify-Mistral Cookbooks
+
+This folder contains cookbooks demonstrating how to leverage Indexify and Mistral's large language models for advanced document processing tasks. These guides provide step-by-step instructions for implementing pipelines for PDF analysis.
```
Comment (diptanu) [inline docs/docs/examples/mistral/README.md:3]: Indexify is an open-source engine for building fast data pipelines for unstructured data(video, audio, images and documents) using re-usable extractors for embedding, transformation and feature extraction. Indexify keeps vectordbs, structured databases(postgres) updated automatically when pipelines produce embedding or structured data.

Applications can query indexes and databases using semantic search and SQL queries.

Project - https://github.com/tensorlakeai/indexify

This folder contains cookbooks demonstrating how to leverage Indexify and Mistral's large language models for building production ready pipelines for document understanding.

File: docs/docs/examples/mistral/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+2. Review the cookbooks to understand the workflow and components.
+3. Follow the step-by-step instructions to implement the pipelines.
+4. Experiment with customizations to tailor the solutions to your specific needs.
+
+## Additional Resources
+
+- [Indexify Documentation](https://docs.getindexify.ai/)
+- [Mistral AI Website](https://mistral.ai/)
+
+We hope these cookbooks help you unlock the power of advanced document processing in your projects!
```
Comment (diptanu) [inline docs/docs/examples/mistral/README.md:53]: Remove this last line

**Review by diptanu — CHANGES_REQUESTED** [review]

File: docs/docs/examples/mistral/README.md (hunk 1 above)
Comment (rishiraj) [reply docs/docs/examples/mistral/README.md]: updated

File: docs/docs/examples/mistral/README.md (hunk 2 above)
Comment (rishiraj) [reply docs/docs/examples/mistral/README.md]: updated

**Pull request merged**